src/components/common/comments/ArticleCommentList.js

```diff
@@ -1,16 +1,26 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
What's this about?

Reply:
@mekzy-o 👆🏽is still here, what is it doing?

Reply:
Eslint was flagging the `like-icon` I used as a button. on researching the error... I saw this: 
```
Enforce onClick is accompanied by at least one of the following: onKeyUp, onKeyDown, onKeyPress. Coding for the keyboard is important for users with physical disabilities who cannot use a mouse, AT compatibility, and screenreader users.
```
Since I didn't want to add any onKeyUp, onKeyDown etc. I had to disable it